doc/using-hipsycl.md

```diff
@@ -1,9 +1,9 @@
-# Using hipSYCL in projects
+# Using Open SYCL in projects
 It is recommended to use the CMake integration for larger projects. See the section on the cmake integration below. Alternatively, `syclcc` can be used directly as a compiler.
 
-## hipSYCL targets specification
+## Open SYCL targets specification
 
-Both `syclcc` and the cmake integration expect a hipSYCL targets specification. This specification defines which compilation flows hipSYCL should enable, and which devices from a compilation flow hipSYCL should target during compilation. In general, it has the form:
+Both `syclcc` and the cmake integration expect a Open SYCL targets specification. This specification defines which compilation flows Open SYCL should enable, and which devices from a compilation flow Open SYCL should target during compilation. In general, it has the form:
```

Comment:
```suggestion
Both `syclcc` and the cmake integration expect an Open SYCL targets specification. This specification defines which compilation flows Open SYCL should enable, and which devices from a compilation flow Open SYCL should target during compilation. In general, it has the form:
```

Reply:
Ahhh, I just search&replaced, haven't thought about that :(

Reply:
I had to look real hard to find that one ;)